### PR TITLE
2.1.3 release

### PR DESCRIPTION
**Pull request by suyograo**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,16 @@
+## 2.1.2 (Feb 16, 2016)
```
Comment (colinsurprenant) [inline CHANGELOG.md]: 2.1.3!?

File: Gemfile.jruby-1.9.lock [hunk 2]
```diff
@@ -298,10 +298,11 @@ GEM
       logstash-codec-plain
       logstash-core (>= 2.0.0.beta2, < 3.0.0)
       stud
-    logstash-input-http (2.1.1)
+    logstash-input-http (2.2.0)
       logstash-codec-plain
       logstash-core (>= 2.0.0.beta2, < 3.0.0)
-      puma (~> 2.11.3)
+      puma (~> 2.16, >= 2.16.0)
```
Comment (colinsurprenant) [inline Gemfile.jruby-1.9.lock]: Puma from 2.11 to 2.16 is that wanted?

Comment (suyograo) [inline Gemfile.jruby-1.9.lock]: Bug fix update, should be safe.. 2.2.1 has no issues

File: CHANGELOG.md (hunk 1 above)
Comment (suyograo) [inline CHANGELOG.md]: huh copy paste

**Comment by colinsurprenant** [comment]
LGTM

**Comment by suyograo** [comment]
Merged with https://github.com/elastic/logstash/commit/f040ab64bfd5e8a276c615068cfa53192d42a464

**Pull request closed without merging**